lib/rouge/lexers/make.rb

```diff
@@ -28,15 +37,25 @@ def initialize(opts={})
 
         rule %r/#.*?\n/, Comment
 
-        rule %r/(export)(\s+)(?=[a-zA-Z0-9_\${}\t -]+\n)/ do
+        rule %r/([-s]?include)((?:[\t ]+[^\t\n #]+)+)/ do
+          groups Keyword, Literal::String::Other
+        end
+
+        rule %r/(ifn?def|ifn?eq)([\t ]+)([^#\n]+)/ do
```

Comment:
Just to confirm: the keywords here are `ifdef`, `ifndef`, `ifeq` and `ifneq`?

Reply:
Correct.